### DISCUSSION TITLE
DeepSpeed

### DISCUSSION THREAD

**Discussion by levicki** [oobabooga/text-generation-webui#3804] [Ideas]
I was wondering if this is something that could be used / integrated?

https://github.com/microsoft/DeepSpeed

**Comment by missionfloyd** [comment]
https://github.com/oobabooga/text-generation-webui/blob/main/docs/DeepSpeed.md

**Comment by levicki** [comment]
@missionfloyd Thanks for the link, I am curious to see what kind of speed improvement can Intel AMX bring compared to plain CPU.
